ScreenManager.c

```diff
@@ -304,6 +304,8 @@ void ScreenManager_run(ScreenManager* this, Panel** lastFocus, int* lastKey, con
                      }
                   }
                }
+            } else if (mevent.bstate & BUTTON3_RELEASED) {
+               ch = KEY_RIGHTCLICK;
```

Comment:
Shouldn't this replicate the `KEY_RECLICK` logic from above?

Reply:
No, we only want to be sure to filter out a right click. Right clicks are not used to select _and_ change as left clicks are (in htop, which is UI-concept wise bit wrong anyways but historic).

Reply:
Ah, k. LGTM.